src/ide/ide.ts

```diff
@@ -2160,27 +1931,39 @@ export let ide = (function () {
 				"text-overflow": "ellipsis clip",
 
 				"text-align": "center",
-				background: "#fff",
+				//background: "#fff",
 			},
 		});
+
+		module.programstate.setCss = function (cls) {
```

Comment:
Maybe call this `setCSSStateClass` or something else more descriptive? When I read `setCss` I'd think of setting styles directly...